.gitignore

```diff
@@ -16,6 +16,7 @@ vendor/
 
 .env*
 /config/share*
+/config/config.yaml
```

Comment:
This will remove the config example.. need to change the filename (e.g. `config.example.yaml`) or remove this

Reply:
Added config.example.yaml